<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsWindow</class>
 <widget class="QWidget" name="settingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>451</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Profile</string>
   </property>
   <widget class="QFrame" name="frame_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="txtUsername">
     <property name="geometry">
      <rect>
       <x>153</x>
       <y>10</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="txtAuthenticationToken">
     <property name="geometry">
      <rect>
       <x>153</x>
       <y>10</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblAuthToken">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Authentication Token</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblInfo">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Your authentication token can be found in your profile on starbowmod.com</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>451</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QFrame" name="frameSearchRange">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblSearchRange">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Search range</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbSearchRange">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string>Narrow</string>
     </property>
     <item>
      <property name="text">
       <string>Narrow</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Normal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Wide</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frameAutostart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblAutostart">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Starbow with Windows</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbAutostart">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yes</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frameLinks">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblChatLinks">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Display links in chat</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbChatLinks">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yes</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frameLanguage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblLanguage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbLanguage">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>English</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frameServer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblServer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Server</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbServer">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string>eros.starbowmod.com</string>
     </property>
     <item>
      <property name="text">
       <string>eros.starbowmod.com</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
